test/e2e/pipeline/pipeline_test.go

```diff
@@ -212,6 +223,27 @@ Waiting for logs to be available...
 		})
 	})
 
+	t.Run("Get list of Taskruns using field-selector from namespace  "+namespace, func(t *testing.T) {
```

Comment:
It would probably be better to test for metadata.name. More than welcome to have a test for both though.

Reply:
Taskruns names are autogenerated so I am not sure how to get that.